worker/run.py

```diff
@@ -88,6 +109,27 @@ def run(self):
 
         return True
 
+    def resume(self):
+        # Report that the bundle is running. We note the start time here for
+        # accurate accounting of time used, since the clock on the bundle
+        # service and on the worker could be different.
+        start_message = {
+            'hostname': socket.gethostname(),
+            'start_time': int(self._start_time),
+        }
+        if not self._bundle_service.resume_bundle(self._worker.id, self._uuid,
+                                                 start_message):
+            print("false!!")
+            return False
+
+        # Start a thread for this run.
+        def resume_run(self):
+            Run._safe_update_run_status(self, 'Running')
+            Run._monitor(self)
+
+        print("resuming bundle {}, container {}".format(self._uuid, self._container_id))
```

Comment:
try to use `logger.info` or `logger.debug` as appropriate instead of print statements